user-guide/Basic_Functionality/Visio/reference/Placeholders_for_variables_in_shape_data_values.md

```diff
@@ -855,6 +855,16 @@ It is possible to override the default behavior of this placeholder for child sh
 
 In a drawing linked to an enhanced service, you can refer to this service with this placeholder.
 
+### \[this reservationID\]
+
+From DataMiner 10.2.8/10.3.0 onwards, you can use this placeholder in shape data or shape text of shapes linked to a booking (e.g. dynamically generated shapes that represent bookings) to retrieve the GUID of the booking. The placeholder can also be nested, for example to retrieve booking properties or resources.
+
+For example:
+
+`[Resource:[reservation:[this reservationID],ResourceID|NodeID=18|],Name]`
+
+`[reservation:[this reservationID],Property=Monitoring]`
+
```

Comment:
Maybe add also a note that this placeholder is often used in combination with shape data 'Options' set to 'AllowInheritance=False|ForcePropertyFromParent'

Reply:
Thanks. Ok to merge.